webpack.config.babel.js

```diff
@@ -165,7 +165,7 @@ const config = {
         ],
       },
       {
-        test: /readme\.md$/,
+        test: /\.md$/,
```

Comment:
Webpack previously could only load Markdown files named `readme.md`. Now it works with any `.md` file.